pkg/valueattribution/valueattribution.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package valueattribution
+
+import (
+	"context"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/satellite/console"
+)
+
+// Error messages
+const (
+	InternalErrMsg         = "It looks like we had a problem on our end. Please try again"
+	NoConnectorIDSetErrMsg = "No ConnectorID set"
+)
+
+// connectorID is a context value key type
+type connectorID string
```

Comment:
I used this file https://github.com/storj/storj/blob/master/satellite/console/auth.go as reference and used the same logic as you mentioned above. Just curious, is there any advantage of having key of type int vs something else?

I started using the context but @coyle suggested to use protobuf to send partner related info instead of context. So I left those functions there just in case if we change direction, otherwise thought of cleaning as a TODO. For now those context functions are not used.
But I will make changes to changing the key type to 'int'

Reply:
Example you point is also using `int`. Int as key is just more compact.

Reply:
ok will change to int to be consistent... also we are not using context to pass key values..... should I still keep it until we decide  or remove all together...let me know what your thoughts are?

Reply:
Removing/leaving context values depends if you can answer my main question. How libuplink user will trigger `ValueAttributeInfo`?

See https://github.com/storj/storj/pull/2091#issuecomment-500821130